codebasin/report.py

```diff
@@ -146,24 +149,34 @@ def clustering(output_name, setmap):
     from scipy.spatial.distance import squareform
 
     # Compute distance matrix between platforms
-    matrix = [[distance(setmap, p1, p2) for p2 in platforms] for p1 in platforms]
+    matrix = [
+        [distance(setmap, p1, p2) for p2 in platforms] for p1 in platforms
+    ]
 
     # Print distance matrix as a table
     lines = []
     lines += ["", "Distance Matrix"]
-    labelled_matrix = [[name] + [("%.2f" % column) for column in matrix[row]]
-                       for (row, name) in enumerate(platforms)]
+    labelled_matrix = [
+        [name] + [f"{column:.2f}" for column in matrix[row]]
+        for (row, name) in enumerate(platforms)
+    ]
     lines += [table([""] + platforms, labelled_matrix)]
 
     # Hierarchical clustering using average inter-cluster distance
-    clusters = hierarchy.linkage(squareform(matrix), method='average')
+    clusters = hierarchy.linkage(squareform(matrix), method="average")
 
     # Plot dendrogram of hierarchical clustering
     fig, ax = plt.subplots()
     hierarchy.dendrogram(clusters, labels=platforms, orientation="right")
     ax.set_xlim(xmin=0, xmax=1)
-    ax.axvline(x=divergence(setmap), linestyle='--', label="Average")
-    plt.text(divergence(setmap), ax.get_ylim()[1], "Average", ha="center", va="bottom")
+    ax.axvline(x=divergence(setmap), linestyle="--", label="Average")
+    plt.text(
```

Comment:
For consistency, can you do `ax.text`?

Functionally I don't think it matters since `plt` just grabs whatever active axis, but just code style

Reply:
Fixed in 3d55e5141708d54c4a74eab0b9db24061afac153.